src/assets/mock-classes.ts

```diff
@@ -0,0 +1,15 @@
+import { Class } from './class';
```

Comment:
Please remove this mock data if it is not used.

Reply:
mock classes removed